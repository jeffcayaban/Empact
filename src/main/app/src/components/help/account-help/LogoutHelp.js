import React from "react";
import {Col, Grid, Row} from "react-bootstrap";
import LogoutLink from '../../../other/images/logout-link.png';

import SuccessfulLogoutDialog from "../../../other/images/successful-logout.png";
import BackButton from "../../common/BackButton";

/**
 * Displays help information for logging out of an account.
 */

class LogoutHelp extends React.PureComponent {

    render() {
        return (
            <div className={'secondaryPageContainerStyle'}>
                <div className={"centerStyle"}>
                    <h2>Logging Out</h2>
                    <p>Understand how to log out of your account.</p>
                </div>
                <div className={"titleContentPadding"}>
                    <Grid className={"divNoPadding"}>
                        <Row>
                            <Col md={12} lg={12}>
                                <div>
                                    <h4 style={{ margin: 0 }}>How to logout</h4>
                                    <hr className={"helpHr"} />

                                    <p>
                                        1. Click on the logout link that is located on the top navigation bar. Clicking
                                        on this link will proceed to logging out of your account.
                                    </p>

                                    <div className={"centerStyle"}>
                                        <img style={{ maxWidth: '35%', minWidth: '350px', align: 'center' }}
                                             src={LogoutLink}
                                             alt="Logout Link"
                                        />
                                    </div>

                                    <p>
                                        2. If the logout is successful then you will be shown a confirmation notification.
                                    </p>

                                    <div className={"centerStyle"}>
                                        <img style={{ maxWidth: '25%', minWidth: '350px', align: 'center' }}
                                             src={SuccessfulLogoutDialog}
                                             alt="Logout Link"
                                        />
                                    </div>
                                </div>
                                <BackButton link={"/help/account"}/>
                            </Col>
                        </Row>
                    </Grid>
                </div>
            </div>
        );
    }
}

export default LogoutHelp;