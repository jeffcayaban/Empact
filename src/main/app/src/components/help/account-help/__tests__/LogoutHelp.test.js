import React from 'react';
import toJson from "enzyme-to-json";
import { shallow } from "enzyme";
import LogoutHelp from "../LogoutHelp";

describe('LogoutHelp', () => {

    it('should render successfully', () => {
        const component = shallow(<LogoutHelp />);
        expect(toJson(component)).toMatchSnapshot();
    });
});
