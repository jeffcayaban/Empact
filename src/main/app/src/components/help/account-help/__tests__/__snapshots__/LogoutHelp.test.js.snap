// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoutHelp should render successfully 1`] = `
<div
  className="secondaryPageContainerStyle"
>
  <div
    className="centerStyle"
  >
    <h2>
      Logging Out
    </h2>
    <p>
      Understand how to log out of your account.
    </p>
  </div>
  <div
    className="titleContentPadding"
  >
    <Grid
      bsClass="container"
      className="divNoPadding"
      componentClass="div"
      fluid={false}
    >
      <Row
        bsClass="row"
        componentClass="div"
      >
        <Col
          bsClass="col"
          componentClass="div"
          lg={12}
          md={12}
        >
          <div>
            <h4
              style={
                Object {
                  "margin": 0,
                }
              }
            >
              How to logout
            </h4>
            <hr
              className="helpHr"
            />
            <p>
              1. Click on the logout link that is located on the top navigation bar. Clicking on this link will proceed to logging out of your account.
            </p>
            <div
              className="centerStyle"
            >
              <img
                alt="Logout Link"
                src="logout-link.png"
                style={
                  Object {
                    "align": "center",
                    "maxWidth": "35%",
                    "minWidth": "350px",
                  }
                }
              />
            </div>
            <p>
              2. If the logout is successful then you will be shown a confirmation notification.
            </p>
            <div
              className="centerStyle"
            >
              <img
                alt="Logout Link"
                src="successful-logout.png"
                style={
                  Object {
                    "align": "center",
                    "maxWidth": "25%",
                    "minWidth": "350px",
                  }
                }
              />
            </div>
          </div>
          <BackButton
            link="/help/account"
          />
        </Col>
      </Row>
    </Grid>
  </div>
</div>
`;
